WindowsServerDocs/networking/technologies/hcn/hcn-declaration-handles.md

```diff
@@ -1,19 +1,20 @@
 ---
 title: RPC context handles for HCN
 description: Information about HCN Network entities, represented using HCN_NETWORK RPC context handles.
+ms.topic: article
 ms.author: daschott
 author: daschott
-ms.date: 11/05/2018
-ms.topic: article
+ms.date: 01/03/2023
+
 ---
 
 # RPC context handles for HCN
 
->Applies to: Windows Server 2022, Windows Server 2019
+Applies to: Windows Server 2022, Windows Server 2019
 
 ## HCN_Network
 
-An HCN Network is an entity which is used to represent a host compute network and its associated system resources and policies. For example, a HCN network will typically consist of a set of metadata (e.g. id, name, type), a virtual switch, a host virtual network adapter (which acts as a default gateway for the network), a NAT instance (if required by the network type), a set of subnet and MAC pools, and any network-wide policies to be applied (e.g. ACLs).
+An HCN Network is an entity that is used to represent a host compute network and its associated system resources and policies. For example, a HCN network will typically consist of a set of metadata (for example, id, name, type), a virtual switch, a host virtual network adapter (that acts as a default gateway for the network), a NAT instance (if required by the network type), a set of subnet and MAC pools, and any network-wide policies to be applied (for example, ACLs).
```

Comment:
The sentence beginning for example, its quite complex. Are you able to split this up or simplify the sentence?